frontend/app/views/info/about.scala.html

```diff
@@ -15,8 +15,8 @@
                 <p>Join before March 31st to become a Founding Member and the price you pay will not rise.</p>
             </div>
             <div class="action-group">
-                <a class="action u-no-top-margin" href="@routes.Joiner.tierList">Join today</a>
-                <a class="action action--secondary u-no-top-margin js-scroll-to" href="#whats-included">
+                <a class="action" href="@routes.Joiner.tierList">Join today</a>
+                <a class="action action--secondary js-scroll-to" href="#whats-included">
```

Comment:
for another PR ```.js-scroll-to``` is not being used in the js - did this slip in when the scroll to functionality was canned?